src/components/Projects/index.tsx

```diff
@@ -0,0 +1,21 @@
+import Dosu from 'components/Projects/Dosu'
+import Echo from 'components/Projects/Echo'
+import Sealcaster from 'components/Projects/Sealcaster'
+import classnames, { margin, space, width, zIndex } from 'classnames/tailwind'
+
+const wrapper = classnames(
+  space('space-y-64', 'md:space-y-96'),
```

Comment:
Can we use `gap` here instead?

Reply:
Yes, but there's no point, I'll need `flex` and `flexDirection` here, it work just the same, but I'll end up with 2 more additional properties: `flex flex-col`